### ISSUE TITLE
Using ngCacheBuster only for IE

### ISSUE THREAD

**Issue by jdubois** [saintmac/angular-cache-buster#8]
We have set up ngCacheBuster for [JHipster](https://github.com/jhipster/generator-jhipster) and now all our request get the added "cache buster" parameter.
While this is good for IE, it is bad for other browsers: this added parameter uses a bit more resources and might be confusing to users.
Could we have an option so that the "cache buster" parameter is only applied on IE?

**Comment by mrpohoda** [comment]
I'd also like to use ngCacheBuster only for IE. Is there any solution?

**Comment by Grv90** [comment]
it not working .js files.

**Comment by saintmac** [comment]
If this is still an issue feel free to create a pull request and I will merge it into the project

**Comment by siddhsuk** [comment]
@saintmac Does this library has any drawback. Can I use only for IE

**Comment by saintmac** [comment]
@siddhsuk No, no drawback. We've been using it in production for more than a year.
For now you cannot use it for IE only, but it's not an issue to use it with other browsers too. If you'd like to submit a pull request with a feature to only use it for IE, feel free to do so, I'll merge it into the main library.
Thanks
